Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -773,15 +771,19 @@ def update_malop_status(malop_guid, status):
     json_body = {malop_guid: api_status}
 
     response = http_request('POST', '/rest/crimes/status', json_body=json_body)
-    if response['status'] != 'SUCCESS':
-        raise Exception('Failed to update malop {0} status to {1}. Message: {2}'.format(malop_guid, status,
-                                                                                        response['message']))
+    status = response.get('status') if response else ''
+    if status != 'SUCCESS':
```

Comment:
```suggestion
    if status.upper() != 'SUCCESS':
```
To be on the safe side...